app/src/test/java/com/kickstarter/libs/utils/DiscoveryParamsUtilsTest.java

```diff
@@ -5,49 +5,59 @@
 import com.kickstarter.libs.RefTag;
 import com.kickstarter.services.DiscoveryParams;
 
-import junit.framework.TestCase;
+import org.junit.Assert;
+import org.junit.Test;
 
-public class DiscoveryParamsUtilsTest extends TestCase {
+public class DiscoveryParamsUtilsTest {
 
+  @Test
   public void testRefTag() {
-    assertEquals(
+    Assert.assertEquals(
```

Comment:
That's how I did it the first time, but milkrun wouldn't allow it, so I changed them